test/alternator/test_metrics.py

```diff
@@ -143,6 +160,18 @@ def test_batch_write_item_histogram(test_table_s, metrics):
         test_table_s.meta.client.batch_write_item(RequestItems = {
             test_table_s.name: items2})
 
+def test_table_batch_write_item_histogram(test_table_s, metrics):
```

Comment:
Nitpick: Above you called the test "test_batch_table" and this one "test_table_batch". I think it's better to be more consistent on the order.

Reply:
good point, I'll keep the test_table for everything